image_generation/lcm_dreamshaper_v7/cpp/src/main.cpp

```diff
@@ -0,0 +1,312 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include <algorithm>
+#include <ctime>
+#include <iostream>
+#include <fstream>
+#include <random>
+#include <string>
+#include <vector>
+#include <cmath>
+#include <filesystem>
+#include <cassert>
+
+#include "openvino/runtime/core.hpp"
+#include "openvino/pass/manager.hpp"
+#include "openvino/core/preprocess/pre_post_process.hpp"
+
+#include "cxxopts.hpp"
+#include "scheduler_lcm.hpp"
+#include "lora.hpp"
+#include "imwrite.hpp"
+
+ov::Tensor randn_tensor(uint32_t height, uint32_t width, bool use_np_latents, uint32_t seed = 42) {
+    ov::Tensor noise(ov::element::f32, {1, 4, height / 8, width / 8});
+    if (use_np_latents) {
+        // read np generated latents with defaut seed 42
+        const char * latent_file_name = "../scripts/np_latents_512x512.txt";
```

Comment:
I don't see this file in the `scripts` folder
Looks like `readNPLatent` option is not applicable to this sample?

Reply:
You can find this file here: https://github.com/openvinotoolkit/openvino.genai/blob/master/image_generation/lcm_dreamshaper_v7/cpp/scripts/np_latents_512x512.txt